host-bin/enter-chroot

```diff
@@ -684,10 +684,19 @@ if [ -n "$NOLOGIN" ]; then
         fi
     fi
 else
-    # Run rc.local
+    # Check and run rc.local
     if [ -n "$firstrun" -a -x "$CHROOT/etc/rc.local" ]; then
+        ( 
+            sleep 10
+            echo "Still running chroot's rc.local..." 2>&1
+            while true 
+            do
+                    sleep 1
+            done
+        ) & forkpid=$! 
```

Comment:
Also, you may want to `addtrap "kill '$forkpid' 2>/dev/null"` and then `undotrap` after chrootcmd, so that a ctrl-c kills the forked process as well.